documentation/decisions/adr/0003-api-language.md

```diff
@@ -0,0 +1,95 @@
+
+### Positive Consequences <!-- OPTIONAL -->
+- We're hopeful that this choice will lead to open source contribution and adoption
+- We can make use of python libraries designed for complex data queries to quickly deliver features and analytics
+
+### Negative Consequences <!-- OPTIONAL -->
+- Will require some context switching when going from work on the API to the front-end
+
+
+## Pros and Cons of the Options <!-- OPTIONAL -->
+
+
+### Python
+
+Python as a scripting language is fast and productive. Web-based applications can be created quickly and the code is highly readable. Python syntax is easy to understand, well-defined and extensible. APIs built over Python are highly scalable and have an unmatched speed.
+
+Because of it's popularity in the open source community, Python makes it easy for open source contributors to support our work. 
+
+- **Pros**
+  - Nava and HHS experience with Python and APIs built with Python
+  - Number of available libraries and frameworks
+  - Robust data science and algorithmic libraries
+  - Free and open source to use
+  - Ideal for data computation and complex database queries
+- **Cons**
+  - Single threaded locking of GIL can make computationally expensive opersation slower than other options
+
+
+### Javascript and Node
+
+Node JS is used to build fast, highly scalable network applications based on an event-driven non-blocking input/output model, single-threaded asynchronous programming. It also suits well for microservice-based products. Such Node.js utility is accomplished by such characteristics as support of a non-blocking input-output and a significant number of connections.
+
+Because of it's popularity in the open source community, Javascript and Node makes it easy for open source contributors to support our work.
+
+- **Pros**
+  - Nava experience with Javascript language and Node based API servers
+  - Fast single threaded but asynchronous event driven operation
+  - Free and open source to use
+  - Same language as most front-end apps requiring less context switching for developers
+  - Could use the same framework for frontend, simplifying infrastructure
+- **Cons**
+  - Not as mature as other options
+  - Responsiveness may be slowed by any CPU intensive computation
+
+
+### Java
+
+Java is widely used for building enterprise-scale web applications as it is one of the most stable languages on the market. Java’s advantages include platform independence, multi-threaded processing, automatic garbage collection, and security.
+
+Because of it's proprietary nature, Java is not as popular with the open source community, which adds some barriers for open source contributors to support our work.
```

Comment:
nit: small grammar fix

```suggestion
Because of its proprietary nature, Java is not as popular with the open source community, which adds some barriers for open source contributors to support our work.
```